### PR TITLE
Add question/answer views to web app

### PR DESCRIPTION
**Pull request by jessieay**
- Use VCR in specs -- YEAH!

screen shot 2016-01-21 at 11 12 58 am

screen shot 2016-01-21 at 11 16 26 am

### CODE REVIEW

File: db/structure.sql [hunk 1]
```diff
@@ -136,7 +136,7 @@ INSERT INTO schema_migrations (version) VALUES ('20150924181500');
 
 INSERT INTO schema_migrations (version) VALUES ('20160112190459');
 
-INSERT INTO schema_migrations (version) VALUES ('20160115181255');
+INSERT INTO schema_migrations (version) VALUES ('20160115203719');
```
Comment (pkarman) [inline db/structure.sql]: what's this about?

**Comment by pkarman** [comment]
@jessieay looks great. Just a question about the migrations.

File: db/structure.sql (hunk 1 above)
Comment (jessieay) [inline db/structure.sql]: ugh I don't know! keeps happening to me.argh. did a git checkout at 8693af5

**Comment by pkarman** [comment]
that travis ci faillure is spurious. I think it was fixed in 467f9489 but maybe that's not on this branch.

@jessieay if you restart that test, I can merge once :green_heart:

**Comment by jessieay** [comment]
@pkarman it's :green_apple: :green_book: :green_heart:  :)

**Pull request merged**